.travis.yml

```diff
@@ -1,15 +1,13 @@
 env:
   global:
     - LC_CTYPE=en_US.UTF-8
-before_install:
-  - brew update
-  - brew install swiftlint
 matrix:
   include:
     - os: osx
       language: objective-c
-      osx_image: xcode7.3
+      osx_image: xcode8.3
 script: 
+  - pod repo update
```

Comment:
I no longer remember :-) I think it was due to the fact that the latest cocoapods (present on Travis CI) does not do the update automatically during other commands, and without the update it was not finding the Nimble version in cocoapods spec. I will check once again how it behaves without the update.